### DISCUSSION TITLE
This project has won JetBrains top 10 project of the decade award

### DISCUSSION THREAD

**Discussion by fakeid30** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#182] [Show and tell]
You can check the tweet [here](https://twitter.com/pycharm/status/1341331157590437888) and them mentioning me [here](https://twitter.com/pycharm/status/1341701435642306561)

We all are pretty happy how this has panned out. 

https://user-images.githubusercontent.com/26362832/103079516-3352b480-45fe-11eb-9d31-9dbabcad6152.mp4

**Comment by Srush23** [comment]
Congratulations!
